vmdb/spec/factories/provider.rb

```diff
@@ -0,0 +1,13 @@
+FactoryGirl.define do
+  factory :foreman_provider, :class => 'ProviderForeman' do
```

Comment:
Should there be a base :provider factory that this factory derives from?  Things like the authentications stuff can just live there instead.